components/script/dom/webidls/OscillatorNode.webidl

```diff
@@ -25,7 +25,7 @@ dictionary OscillatorOptions : AudioNodeOptions {
  Constructor (BaseAudioContext context, optional OscillatorOptions options)]
 interface OscillatorNode : AudioScheduledSourceNode {
 //  [SetterThrows]
```

Comment:
We need to uncomment this line as well, so we can throw an `InvalidStateError` exception when trying to set the type to `OscillatorType::Custom`, as [specified](https://webaudio.github.io/web-audio-api/#dom-oscillatornode-type)